src/layers/models/layersManager.ts

```diff
@@ -43,79 +42,64 @@ class LayersManager {
     return requestedLayer;
   }
 
-  public async addLayer(layerRequest: ILayerPostRequest): Promise<void> {
-    const editJson = (jsonDocument: IMapProxyJsonDocument): IMapProxyJsonDocument => {
-      this.addNewCache(jsonDocument, layerRequest);
-
-      return jsonDocument;
-    };
-
-    await this.configProvider.updateJson(editJson);
-  }
+  public async getCacheName(layerName: string, cacheType: string): Promise<ICacheName> {
+    const errorMsg = `Layer ${layerName} or fitting cache do not exist`
```

Comment:
please move this errorMsg definition to were it's beeing used

Reply:
moved